lib/beaker/dsl/install_utils/pe_defaults.rb

```diff
@@ -32,7 +32,7 @@ module PEDefaults
             'puppetbin'        => '/opt/puppet/bin/puppet',
             'puppetbindir'     => '/opt/puppet/bin',
             'puppetsbindir'    => '/opt/puppet/sbin',
-            'privatebindir'    => '/opt/puppetlabs/puppet/bin',
+            'privatebindir'    => '/opt/puppet/bin',
```

Comment:
This winds up flipping the desired values. Looks like I gave you a bum steer @kevpl